5calls/app/src/main/java/org/a5calls/android/a5calls/net/FiveCallsApi.java

```diff
@@ -130,7 +134,15 @@ public void unregisterContactsRequestListener(ContactsRequestListener contactsRe
     }
 
     public void getIssues() {
-        buildIssuesRequest(GET_ISSUES_REQUEST, mIssuesRequestListeners);
+        String url = GET_ISSUES_REQUEST;
+        
+        // Include state parameter if we have it stored
+        String state = AccountManager.Instance.getState(mContext);
+        if (!TextUtils.isEmpty(state)) {
+            url += "?state=" + URLEncoder.encode(state);
```

Comment:
can `?state=` be a constant like GET_ISSUES_REQUEST, or could we have GET_ISSUES_REQUEST_WITH_STATE that includes it?